ui/components/component-library/banner-tip/banner-tip.constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum BannerTipLogoType {
+  greeting = 'greeting',
+  chat = 'chat',
```

Comment:
Per slack chat we want all of our enums in the extension to be PascalCase https://consensys.slack.com/archives/G1L7H42BT/p1677617615825159?thread_ts=1677616875.751329&cid=G1L7H42BT

```suggestion
  Greeting = 'greeting',
  Chat = 'chat',
```

Make sure to update the docs